src/lib/hooks/use-best-quote/utils/index-quote.ts

```diff
@@ -81,24 +76,11 @@ export async function getIndexQuote(
 
     let gasCosts = BigInt('0')
     let gasLimit = BigInt('0')
-    let gasPrice = BigInt(request.gasPrice.toString())
+    let gasPrice = tx.gasPrice
     const estimate = res.rawResponse.estimate
     const isLifiQuote = res.type === 'lifi'
-    if (isLifiQuote) {
-      gasLimit = BigInt(estimate?.gasCosts[0].limit ?? '0')
-      gasPrice = BigInt(estimate?.gasCosts[0].price ?? '0')
-    } else {
-      // Right now this should only be ic21
-      const defaultGas = 120_000
-      const defaultGasEstimate = BigInt(defaultGas)
-      const gasEstimatooor = new GasEstimatooor(provider, defaultGasEstimate)
-      // We don't want this function to fail for estimates here.
-      // A default will be returned if the tx would fail.
-      const canFail = false
-      const gasEstimate = await gasEstimatooor.estimate(tx, canFail)
-      console.log(gasEstimate.toString(), 'gasLimit')
-      gasLimit = gasEstimate
-    }
+    gasLimit = BigInt(estimate?.gasCosts[0].limit ?? '0')
+    gasPrice = BigInt(estimate?.gasCosts[0].price ?? '0')
     gasCosts = gasPrice * gasLimit
```

Comment:
interesting - it seems like this works with the string values, but should they be numbers?